### PR TITLE
Change MongoDB CDC guide to use connector

### LINKED ISSUE #1900: Document: feat(cdc): Add MongoDB CDC Source
This issue tracks the documentation update needed for the merged PR #14966.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/14966
Source PR Merged At: 2024-02-29T07:45:16Z

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  - Change MongoDB CDC guide to detail connector usage instead of Debezium+Kafka process

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/14966

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1900

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by emile-00** [comment]
@StrikeW Please let me know if there is additional info like connector parameters or MongoDB setup configs we should include. Thanks!

**Review by ShanlanLi — APPROVED** [review]
LGTM

File: docs/guides/ingest-from-mongodb-cdc.md [hunk 1]
```diff
@@ -9,79 +9,52 @@ keywords: [mongodb cdc, data source]
...
-`<mongodb host>:27017` refers to a comma-separated list of hostname and port pairs (in the form 'host' or 'host:port') of the MongoDB servers in the replica set.
+### Connector parameters
+
+Unless specified otherwise, the fields listed are required. Note that the value of these parameters should be enclosed in single quotation marks.
 
-`<message queue host>:29092` refers to the bootstrap servers for the Kafka cluster that will store the database schema history topic.
+|Field|Notes|
+|---|---|
+|mongodb.url| The URL of MongoDB. |
+|collection.name| The collection you want to ingest data from. Use the format `db_name.collection_name` to specify which database the collection is located in. |
```
Comment (StrikeW) [inline docs/guides/ingest-from-mongodb-cdc.md]: we support specify a comma-separated list of regular expressions in the `collection.name`. For example user can specify `'foo.*,bar.*'` to capture collections in the `foo` and `bar` databases.

Comment (emile-00) [reply docs/guides/ingest-from-mongodb-cdc.md]: I've edited the description accordingly. Let me know if it needs to be fixed. Thanks!

**Pull request merged**